bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+set -vx
+
+bundle install
+
+# Do any other automated setup that you need to do here
```

Comment:
If it's part of the development workflow, it's kinda weird for me as I'm using... zsh (don't shoot, I'm good people 😄).

I'll leave it, I guess a little ceremony couldn't hurt.